docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -82,214 +72,347 @@ To enable your application to use the image util functionality:
+
+3. Execute the transformation using `image_util_transform_run()`:
+
    ```
-<a name="crop"></a>
-## Cropping an Image
+   ret = image_util_transform_run(transform_h, (media_packet_h)src_packet,
+                                (image_util_transform_completed_cb)completed_callback,
+                                user_data);
+   ```
+
+   > **Note**
+   >
+   > The image format has no effect on the transformation. If the color space is YUV, then the width and height of the target image must be multiples of eight. This restriction does not apply to the RGB images.
+
+4. Handle the transformation results in `image_util_transform_completed_cb()`, which is invoked after the transformation is complete.
+
+5. After the transformation is complete, destroy the transformation handle using `image_util_transform_destroy()`:
+
+   ```
+   ret = image_util_transform_destroy(transform_h);
+   ```
+
+<a name="crop_image"></a>
+## Cropping Image
 
 To crop an image:
 
-1. Create a transformation handle using the `image_util_transform_create()` function:
+1. Create a transformation handle using `image_util_transform_create()`:
 
    ```
-   transformation_h handle;
-   ret = image_util_transform_create(&handle);
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
    ```
 
-2. Set the crop area using the `image_util_transform_set_crop_area()` function:
+2. Set the crop area using `image_util_transform_set_crop_area()`:
 
    ```
-   ret = image_util_transform_set_crop_area(handle, start_x, start_y, end_x, end_y);
+   ret = image_util_transform_set_crop_area(transform_h, start_x, start_y, end_x, end_y);
    ```
 
-3. Execute the transformation using the `image_util_transform_run()` function:
+3. Execute the transformation using `image_util_transform_run()`:
 
    ```
-   ret = image_util_transform_run(handle, src,
-                                  (image_util_transform_completed_cb)completed_callback,
-                                  user_data);
+   ret = image_util_transform_run2_async(transform_h, src_image,
+                                        (image_util_transform_completed2_cb)completed_callback,
+                                        user_data);
    ```
 
    > **Note**
    >
-   > - Because of a YUV restriction, and because the crop start position can be set arbitrarily, the cropped image width and height must be even.
-   > - If hardware acceleration is enabled, you can execute 2 more image transformations using the same transformation handle.
+   > Because of a YUV restriction, and because the crop start position can be set arbitrarily, the cropped image width and height must be even.
```

Comment:
This sentence seems to be a little wordy, can we rephrase it:

As there is a YUV restriction and the crop start position can be set arbitrarily, the cropped image width and height must be even. 

Please verify.